### PR TITLE
Bringing into existing release process

### PR DESCRIPTION
**Pull request by r2liquibase**
Hi, @fbiville ! In anticipation of releasing this extension soon (thank you!!!), I'm trying to align this with our existing release process for the other extensions. It's not perfect, but I'll take consistent 'good enough' over stochastic perfect. Ha!

### CODE REVIEW

**Comment by r2liquibase** [comment]
I did the `release:prepare` step and ran into this:

```
    [INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ liquibase-neo4j ---
    [INFO] Changes detected - recompiling the module!
    [INFO] Compiling 7 source files to /tmp/build/target/classes
    [INFO] -------------------------------------------------------------
    [ERROR] COMPILATION ERROR :
    [INFO] -------------------------------------------------------------
    [ERROR] Could not load processor class file due to 'org/neo4j/annotations/service/ServiceAnnotationProcessor has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0'.
    [INFO] 1 error
    [INFO] -------------------------------------------------------------
    [INFO] ------------------------------------------------------------------------
    [INFO] BUILD FAILURE
    [INFO] ------------------------------------------------------------------------
    [INFO] Total time:  28.170 s
    [INFO] Finished at: 2021-03-30T13:54:08Z
    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project liquibase-neo4j: Compilation failure
    [ERROR] Could not load processor class file due to 'org/neo4j/annotations/service/ServiceAnnotationProcessor has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0'.
    [ERROR]
    [ERROR] -> [Help 1]
    [ERROR]
    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
    [ERROR] Re-run Maven using the -X switch to enable full debug logging.
    [ERROR]
    [ERROR] For more information about the errors and possible solutions, please read the following articles:
    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

It's probably shown in the build failure above. Can you look at that and see where I messed up? Apologies for creating more work! But, the end goal is automated releases; I've got that one. Thanks!

**Review by fbiville — COMMENTED** [review]
I'll investigate the other failure

File: pom.xml [hunk 1]
```diff
@@ -84,49 +95,163 @@
...
-                        <!-- disables Neo4j annotation processors -->
-                        <arg>-proc:none</arg>
-                    </compilerArgs>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                    <optimize>true</optimize>
+                    <debug>true</debug>
+                    <encoding>${project.build.sourceEncoding}</encoding>
+                    <!-- disables Neo4j annotation processors -->
+                    <arg>-proc:none</arg>
```
Comment (fbiville) [inline pom.xml:112]: this needs to be wrapped in `compilerArgs`:
```
    <compilerArgs>
        <arg>-proc:none</arg>
    </compilerArgs>
```

File: pom.xml [hunk 2]
```diff
@@ -84,49 +95,163 @@
...
+                <configuration>
+                    <skip>false</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <author>false</author>
+                    <doctitle>Liquibase CDI ${project.version} API</doctitle>
```
[the remark this answers is not in the document]
Comment (r2liquibase) [reply pom.xml:169]: It is. I know it's weird, but that's what we use for all the other extensions.

**Comment by r2liquibase** [comment]
OK! Thanks @fbiville !!! It was the `compilierArgs`. Ugh...I totally saw that when I was moving over but still missed it. Bad, Robert!

**Comment by r2liquibase** [comment]
And, we're good!

I created a 0.1 release, pushed to Maven Central, and it showed up. 

I cleaned up by dropping the release from Maven Central and deleting th 0.1 tag. 

Lastly, I set the version to 4.3.3-SNAPSHOT to align with Liquibase.

**Comment by fbiville** [comment]
Thanks a lot, I'll merge this tomorrow!

**Pull request merged**